R/viz.R

```diff
@@ -1608,6 +1613,9 @@ prettyOncoplot = function(maftools_obj,
   if (! is.null(custom_colours)){
     colours = custom_colours
   }
+
+  colours[["Silent"]] = "#A020F0"
```

Comment:
Maybe this should be added elsewhere to the function that handles all colors for mutations? That way, we store all color assignments in one place